lib/tapioca/config.rb

```diff
@@ -35,6 +35,8 @@ def outpath
     DEFAULT_GEMDIR = T.let("#{DEFAULT_RBIDIR}/gems", String)
     DEFAULT_TODOSPATH = T.let("#{DEFAULT_RBIDIR}/todo.rbi", String)
 
+    DSL_COMPILERS_PATH = T.let("#{__dir__}/compilers/dsl", String)
+
```

Comment:
Initially this path was being generated in `generator.rb` which gave me a fun headache trying to determine why compilers weren't being found after the copy/pasta. 

Decided that since this is a static path, this is probably the best place for it to live.